src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -19,14 +19,14 @@ ConversationStore conversationStore = ConversationStore.getInstance();
 <body>
 
   <div id="container">
-    <h1>Activity
+    <h1>Activity Feed
```

Comment:
Would be nice to have some `margin-top` for this element, and maybe make it `<h3>`?